components/SeoHead.js

```diff
@@ -0,0 +1,89 @@
+import NextHead from 'next/head'
+import { Component } from 'react'
+
+const defaultTitle = 'styled-components'
+const defaultDescription = 'Visual primitives for the component age. Use the best bits of ES6 and CSS to style your apps without stress 💅'
+const defaultImage = 'https://www.styled-components.com/static/atom.png'
+
+// const CreateMarkup = text => ({ __html: text })
+
+
+export default class SeoHead extends Component {
+  render() {
+    const props = this.props
+    return (
+      <NextHead>
+        <title>
+          {props.title || defaultTitle}
+        </title>
+
+        <meta name="description" content={props.description || defaultDescription} />
```

Comment:
Personally, I'd prefer destructuring here, since `props.*` is not really useful and `this.props.*` is obviously a tad long.